activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -61,9 +61,7 @@ def self.initialize_i18n(app)
       directories = watched_dirs_with_extensions(reloadable_paths)
       reloader = app.config.file_watcher.new(I18n.load_path.dup, directories) do
         I18n.load_path.keep_if { |p| File.exist?(p) }
-        I18n.load_path |= reloadable_paths.flat_map(&:existent)
-
-        I18n.reload!
+        I18n.load_path = I18n.load_path | reloadable_paths.flat_map(&:existent)
```

Comment:
Why expand `|=` to `= ... | ...`? Doesn't cause reload by `|=`?

Reply:
Sorry I was expanding it out cause `|=` was confusing for me. Fixed :)